docs-samples/V4/JS/send-messages/app.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Botbuilder v4 SDK - Send messages
+ * 
+ * This bot demonstrates how to send a simple text message.
```

Comment:
Elaborate on this saying that you can send a simple text message with "context.sendActivity()". However, if you want to send a message with options like speech and inputHints then use the "MessageFactory.Text()" … we should call out the difference and explain why they would want to use the latter...